portal/code/cx-portal-shared-components/src/components/basic/Dropzone/components/InputContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, useTheme } from '@mui/material'
+import { CustomIcon } from '../../CustomIcons'
+import { Typography } from '../../Typography'
+
+interface InputContentProps {
+  title: string,
+  subTitle: string,
+}
+
+export const InputContent = ({title,
+  subTitle}: InputContentProps) => {
+  const theme = useTheme()
+  const { icon01 } = theme.palette.icon
+
+  return (
+    <Box>
+      <CustomIcon name={"cloud-upload"} fillColor={icon01} size={64} />
+      <Typography 
+        variant="h4" 
+        sx={{ 
+              display: 'block', 
+              fontFamily: theme.typography.body1.fontFamily,
+              marginTop: '29.5px'
```

Comment:
Better to use the theme spacing (the base is 8px). If you use `marginTop: 4`, it would mean 4 * 8px = 32px

Reply:
Thanks! It's done